requirements.docs.txt

```diff
@@ -0,0 +1,41 @@
+# GRPC Python setup requirements
+# The requirements listed below are used to generated API reference documentation.
+absl-py==1.4.0
```

Comment:
Also, this is looks more like a `requirements.lock` file than a `requirements.txt` file. In general, the pattern is:

1. provide a `requirements.txt` file that only includes direct dependencies and doesn't pin specific versions but instead indicates as wide a version range as wide as possible
2. generate a `requirements.lock` from the `requirements.txt` file and doing `pip install -r requirements.txt && pip freeze > requirements.lock` from a fresh venv
3. Repeat step 2 manually every once in a while when you want to update.

Reply:
Thanks for explaining that! I didn't realize there was a difference. I'll rename it to `requirements.docs.lock`.

Reply:
Can we please also add the corresponding `requirements.docs.txt` that only includes direct dependencies and uses version ranges that are as broad as possible?